digit-health-services/health-sync-service/src/main/java/org/digit/health/sync/service/compressionService/Compression.java

```diff
@@ -0,0 +1,19 @@
+package org.digit.health.sync.service.compressionService;
+
+import org.digit.health.sync.service.compressionService.compressor.Compressor;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStream;
+
+public class Compression {
```

Comment:
Remove the compressionService directory. Rename the class to CompressionService

Reply:
Done